vagrant/playbooks/source-install.yml

```diff
@@ -26,34 +26,6 @@
           "You must update Ansible to at least 2.10 to use this version of Pulp 3 Installer."
       when: "'fips' in ansible_fqdn"
 
-    - name: Clean built collection
-      file:
-        state: absent
-        path: "{{ item }}"
-      with_fileglob:
-        - "../../pulp-pulp_installer-*.tar.gz"
-      delegate_to: localhost
-      become_user: "{{ lookup('env', 'USER') }}"
```

Comment:
you can use it to remove `~/.ansible/collections/ansible_collections/pulp/`

Reply:
Yeah, I was leaning against doing this. But now that I think about it, it is for the best.

Reply:
@fao89 I added logic to delete the pulp_installer collection specifically (leaves pulp_squeezer installed.)